setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="Open edX",
-    version="0.3",
+    version="0.3.1",
```

Comment:
Ha :-) I've been going back and forth as to how to do this, so I agree that backing out the change makes sense.